packages/frontend/src/components/PageLayout/Accounts/useAccounts.tsx

```diff
@@ -174,7 +174,9 @@ export const useAccounts = ({
   const accounts: AccountMenuItem[] = [
     ...(endUser ? [endUserAccount] : []),
     ...otherUsersAccounts,
-    ...(organizationAccounts.length > 1 ? [...organizationAccounts, allOrganizationsAccount] : organizationAccounts),
+    ...(organizationAccounts.length > 1 && organizationAccounts.length <= 20
```

Comment:
There is a caveat here that is easily missed: The threshold of 20 can be easily hit if the organizationAccounts including their subParties >= 20, meaning the count check should take this into account.

I think `flattenParties(organizationAccounts).length < 20` should work.

Reply:
`flattenParties ` from `useDialogs`....